config/install/media_entity.bundle.images_media_bundle.yml

```diff
@@ -0,0 +1,16 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - media_entity_image
+id: images_media_bundle
+label: 'Images media bundle'
+description: 'Media bundle for Images'
```

Comment:
This should be a more sensible description. Try to be less technical. Users may not even know what the term "bundle" means.